packages/react-server/src/ReactFizzServer.js

```diff
@@ -308,7 +312,10 @@ function createTask(
     legacyContext,
     context,
     assignID,
-  };
+  }: any);
+  if (__DEV__) {
+    task.componentStack = currentTaskInDEV ? task.componentStack : null;
```

Comment:
Why do you check `currentTaskInDEV` before reading from `task`??

Reply:
This looks like a mistake. You're assigning `task.componentStack = task.componentStack`

Reply:
Oops. Yea. Looks like I should add a test that tests component stacks when there's a suspended boundary inbetween.